packages/wonder-blocks-popover/components/popover.md

```diff
@@ -0,0 +1,265 @@
+#### Example: Opening a popover using a trigger element
+
+You always need to add a trigger element inside the Popover to control when and/or from where to open the popover dialog.
+
+```jsx
+const {StyleSheet} = require("aphrodite");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {View} = require("@khanacademy/wonder-blocks-core");
+
+const styles = StyleSheet.create({
+    example: {
+        alignItems: "center",
+    }
+});
+
+<View style={styles.example}>
+    <Popover
+        onClose={()=> console.log('popover closed!')}
+        content={
+            <PopoverContent
+                closeButtonVisible
+                title="Title"
+                content="Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip commodo."
+                image={<img src="/illustration.svg" width={288} height={200} />}
+            />
+        }
+    >
+        {({open}) => (
+            <Button onClick={open}>Open default popover</Button>
+        )}
+    </Popover>
+</View>
+```
+
+#### Example: Opening a popover programmatically
+
+Sometimes you'll want to trigger a popover programmatically. This can be done by
+setting the `opened` prop to `true`. In this situation the `Popover` is a
+controlled component. The parent is responsible for managing the opening/closing
+of the popover when using this prop. This means that you'll also have to update `opened` to false in response to the onClose callback being triggered.
+
+```jsx
+const {StyleSheet} = require("aphrodite");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const Color = require("@khanacademy/wonder-blocks-color").default;
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {default: Icon, icons} = require("@khanacademy/wonder-blocks-icon");
+const {Strut} = require("@khanacademy/wonder-blocks-layout");
+const Spacing = require("@khanacademy/wonder-blocks-spacing").default;
+const {HeadingSmall, LabelLarge} = require("@khanacademy/wonder-blocks-typography");
+
+const customIcon = {
+    small: "M6.92820 0L13.85640 4L13.85640 12L6.92820 16L0 12L0 4Z",
+};
+
+const styles = StyleSheet.create({
+    example: {
+        paddingTop: 300
```

Comment:
SUPER NIT: Would be nice to use `Spacing` constants in doc examples (and multiples thereof).